packages/block-library/src/quote/index.js

```diff
@@ -108,7 +108,7 @@ export const settings = {
 				blocks: [ 'core/paragraph' ],
 				transform: ( { value, citation } ) => {
 					const paragraphs = [];
-					if ( value ) {
+					if ( value && value !== '<p></p>' ) {
```

Comment:
I recall a discussion about this being non-ideal. Is this a "gotta get worse before getting better" thing? Do we have an issue to track the improvements?

Reply:
See #10763.

Reply:
@youknowriad After the enter "fix", do we still need these changes?